lib/features/family/features/reflect/presentation/widgets/recommendations_list_widget.dart

```diff
@@ -48,6 +49,14 @@ class RecommendationsListWidget extends StatelessWidget {
             ),
           ),
         ),
+        if (uiModel.isNotLoggedInparent && !uiModel.showActsOfService)
+          Padding(
+            padding: const EdgeInsets.symmetric(vertical: 24, horizontal: 24),
+            child: TitleSmallText(
+              "\"We're still working on a way for invited parents to make donations here. Stay tuned—it's coming soon!\"",
```

Comment:
Should this say "invited". Technically if you invite me and on my phone I start the game and add you as a player. You would see this, but you are not "the invited" one.